sdk/python/kfp/dsl/_component_bridge.py

```diff
@@ -11,91 +11,187 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Function for creating ContainerOp instance."""
 
```

Comment:
When reviewing this file, better check the "Hide whitespace changes" option.
The file previously used 4 spaces indentation, I changed it to 2 spaces to be compliant with Google style.